client/python/modeldb/utils/ConfigUtils.py

```diff
@@ -1,68 +1,71 @@
 import sys
-import yaml
+import json
 import ConfigConstants as constants
 
 class ConfigReader(object):
     def __init__(self, filename):
         # TODO: need to deal with errors here
-        self.config =  yaml.load(file(filename, 'r'))
+        self.config =  json.load(file(filename, 'r'))
         self.validate_config()
 
     def validate_config(self):
         # TODO: what all do we expect to be here?
-        if constants.PROJECT_KEY not in self.config:
+        if 'project' not in self.config:
             print "Project must be defined in config file. Cannot proceed."
             sys.exit(-1)
-        if constants.GIT_USERNAME_KEY not in self.config:
```

Comment:
We can keep constants, but will need to define new ones so that they
- match `syncer.json`'s keys
- allow for nesting, which syncer.json currently uses.

Basically, if we use constants, we need to introduce a separation of _input_ and _output_ constants. This will allow the python code to continue to ingest the same `syncer.json` file that Scala uses.

Would introducing a bunch of new input constants be ok with you?